spec/features/results_spec.rb

```diff
@@ -143,6 +183,76 @@
         expect(results_page.subjects_filter).to have_content("Primary")
         expect("Biology").to appear_before("Primary")
       end
+
+      context "with parameters" do
+        let(:sort) { "-provider.provider_name,-name" }
+
+        let(:results_page_request) do
+          stub_request(:get, default_url)
+            .with(query: {
+              "filter[vacancies]" => "true",
```

Comment:
We've got `base_parameters` that we could use here. I agree with @danielburnley about maybe extracting that for reuse. I'm just starting on another 'do the filtering' card so I'm happy to address that there if it's a pain to pull out in this PR.